docs/source/configure.rst

```diff
@@ -0,0 +1,504 @@
+
+* :attr:`~CoreSection.host_blocks`
+* :attr:`~CoreSection.nick_blocks`
+
+
+Logging
+=======
+
+Sopel's outputs are redirected to a file named ``<base>.stdio.log``, located in
+the **log directory**, which is configured by :attr:`~CoreSection.logdir`.
+
+The ``<base>`` prefix refers to the configuration's
+:attr:`~sopel.config.Config.basename` attribute.
+
+It uses the built-in :func:`logging.basicConfig` function to configure its
+logs with the following arguments:
+
+* ``format``: set to :attr:`~CoreSection.logging_format` if configured
+* ``datefmt``: set to :attr:`~CoreSection.logging_datefmt` if configured
+* ``level``: set to :attr:`~CoreSection.logging_level`, default to ``WARNING``
+  (see the Python documentation for `available logging levels`__)
+
+.. __: https://docs.python.org/3/library/logging.html#logging-levels
+
+Example of configuration for logging:
+
+.. code-block:: ini
+
+   [core]
+   logging_level = INFO
+   logging_format = [%(asctime)s] %(levelname)s - %(message)s
+   logging_datefmt = %Y-%m-%d %H:%M:%S
+
+.. versionadded:: 7.0
+
+   Configuration options ``logging_format`` and ``logging_datefmt`` have been
+   added to extend logging configuration.
+
+.. versionchanged:: 7.0
+
+   The log filename has been renamed from ``stdio.log`` to ``<base>.stdio.log``
+   to prevent conflict when running more than one instance of Sopel.
+
+Log to a Channel
+----------------
+
+It is possible to send logs to an IRC channel, by configuring
+:attr:`~CoreSection.logging_channel`. By default, it uses the same log level,
+format, and date-format parameters as console logs. This can be overridden
+with these settings:
+
+* ``format`` with :attr:`~CoreSection.logging_channel_format`
+* ``datefmt`` with :attr:`~CoreSection.logging_channel_datefmt`
+* ``level`` with :attr:`~CoreSection.logging_channel_level`
+
+Example of configuration to log errors only in the ``##bot_logs`` channel:
+
+.. code-block:: ini
+
+   [core]
+   logging_level = INFO
+   logging_format = [%(asctime)s] %(levelname)s - %(message)s
+   logging_datefmt = %Y-%m-%d %H:%M:%S
+   logging_channel = ##bot_logs
+   logging_channel_level = ERROR
+   logging_channel_format = %(message)s
+
+.. versionadded:: 7.0
+
+   Configuration options ``logging_channel_level``, ``logging_channel_format``
+   and ``logging_channel_datefmt`` has been added to extend logging
+   configuration.
+
+Raw Logs
+--------
+
+It is possible to store raw logs of what Sopel receives and sends by setting
+the flag :attr:`~CoreSection.log_raw` to true.
+
+In that case, IRC messages received and sent are stored into a file named
+``<base>.raw.log``, located in the log directory.
+
+The ``<base>`` prefix refers to the configuration's
+:attr:`~sopel.config.Config.basename` attribute.
+
+.. versionchanged:: 7.0
+
+   The log filename has been renamed from ``raw.log`` to ``<base>.raw.log``
+   to prevent conflict when running more than one instance of Sopel.
+
+
+Misc
```

Comment:
Let's call this section either "Misc." with the dot (it's an abbreviation), or "Miscellaneous". Or "Other". That works too (and it's short).

Reply:
```suggestion
Other
```